config/app.php

```diff
@@ -1,7 +1,6 @@
 <?php
 
 return [
-
```

Comment:
Hmm there seems to be a lot of whiteline issues with the config files that don't match with the laravel/laravel skeleton. Can you re-check all of those so they match the skeleton ones?

Reply:
Maybe it's your IDE formatter that re-parses these?